openwisp_controller/config/migrations/0056_ow_zt-to-global.py

```diff
@@ -0,0 +1,18 @@
+from django.db import migrations
```

Comment:
Please rename this file to `0056_ow_zt_to_global.py`, only underscores, no dashes!